### ISSUE TITLE
Typo in class check IntercomFcmInstanceIdService

### ISSUE THREAD

**Issue by NicolasBonduel** [intercom/intercom-android#342]
## Version info
  - Intercom for Android version: 3.2.0
  - Android OS version: *
  - Android Device: *

## Expected behavior

In the `intercom-sdk-base`, within the `IntercomPushManager` class, there is a check to know if FCM and/or GCM is implemented. Only the ones present are detected and being used as Push notification tools.

## Actual behavior

I have been trying to implement FCM for a bit when I noticed a type inside the `getFcmInstanceIdClass` method. Indeed, it checks for the `IntercomFcmInstanceIDService` library while in fact the library is named `IntercomFcmInstanceIdService` (with a lowercase D in ID).

## Steps to reproduce
 1. Install FCM
 2. Install intercom-android
 3. Use breakpoints inside the `IntercomPushManager` class.

![screenshot from 2017-05-05 14-24-33](https://cloud.githubusercontent.com/assets/6507454/25745300/9e488792-319e-11e7-8e44-c8216dbace7c.png)

**Comment by ronocod** [comment]
Thanks for highlighting this, it mostly just affects logging for apps using our FCM plugin. We'll have a fix for this in the next release.

**Comment by NicolasBonduel** [comment]
From my understanding of the library, it doesn't affect only the logging. the getFcmInstanceIdClass method is called to check if the FCM library is installed. If the library thinks FCM is not installed, then it doesn't send the Firebase token to intercom, therefore the notifications stop working.

```JAVA

    static IntercomPushManager.IntercomPushIntegrationType getInstalledModuleType() {
        IntercomPushManager.IntercomPushIntegrationType integrationType = IntercomPushManager.IntercomPushIntegrationType.NONE;
        boolean fcmInstalled = fcmModuleInstalled();
        boolean gcmInstalled = gcmModuleInstalled();
        if(fcmInstalled && gcmInstalled) {
            TWIG.internal("Both FCM and GCM are installed");
            integrationType = IntercomPushManager.IntercomPushIntegrationType.BOTH;
        } else if(fcmInstalled) {
            TWIG.internal("FCM is installed");
            integrationType = IntercomPushManager.IntercomPushIntegrationType.FCM;
        } else if(gcmInstalled) {
            TWIG.internal("GCM is installed");
            integrationType = IntercomPushManager.IntercomPushIntegrationType.GCM;
        }

        return integrationType;
    }

    private static boolean gcmModuleInstalled() {
        return getGcmListenerClass() != null && getGcmRegistrationServiceClass() != null;
    }

    private static boolean fcmModuleInstalled() {
        return getFcmMessengerClass() != null && getFcmInstanceIdClass() != null;
    }
```

**Comment by ronocod** [comment]
The result of that method doesn't affect Firebase token registration, we handle it automatically regardless of what that method returns.

**Comment by ronocod** [comment]
Fixed in 3.2.2.
